packages/kbn-discover-utils/src/components/custom_control_columns/types.ts

```diff
@@ -16,9 +16,8 @@ export interface RowControlRowProps {
   record: DataTableRecord;
 }
 
-export interface RowControlProps {
+export interface RowControlProps extends EuiButtonIconProps {
```

Comment:
I would not change this type as it can be rendered also as `EuiContextMenuItem` https://github.com/elastic/kibana/blob/8a89c8576df104c16ae3abf39b9db766e56d4e3b/packages/kbn-unified-data-table/src/components/custom_control_columns/additional_row_control/row_menu_control_column.tsx#L46